src/codeu/chat/server/Model.java

```diff
@@ -67,10 +69,14 @@ public int compare(Time a, Time b) {
   private final Store<Time, Message> messageByTime = new Store<>(TIME_COMPARE);
   private final Store<String, Message> messageByText = new Store<>(STRING_COMPARE);
 
+  private HashMap<Uuid, InterestSet> interestMap = new HashMap<Uuid, InterestSet>(100);
```

Comment:
You can omit the `100` initial size here - looks like a premature optimization to me.